test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -681,6 +802,7 @@ func createPods(namespace string, cpo *createPodsOp, clientset clientset.Interfa
 	if err != nil {
 		return err
 	}
+	klog.Infof("Creating %d pods in namespace %q", cpo.Count, namespace)
```

Comment:
b.Info is easier to deal with on debugging tools

Reply:
the logs get truncated, not sure if there is an option to prevent that.